From 11c819c9424892c9ea57a5ef5e263867915dbc29 Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Thu, 2 Feb 2012 06:21:31 +0100
Subject: [PATCH] man: document that we support tcpwrappers only for access
 control

We do not support, and explicitly never want to support environment
variable settings and suchlike in tcpwrappers.

https://bugs.freedesktop.org/show_bug.cgi?id=45143
(cherry picked from commit 9f056f4087b9b0c88decc33c32078f8503efa46e)
---
 man/systemd.exec.xml |   11 ++++++++---
 1 files changed, 8 insertions(+), 3 deletions(-)

diff --git a/man/systemd.exec.xml b/man/systemd.exec.xml
index 230c4a3..f96bb3d 100644
--- a/man/systemd.exec.xml
+++ b/man/systemd.exec.xml
@@ -634,14 +634,19 @@
                                 conjunction with socket-activated
                                 services, and stream sockets (TCP) in
                                 particular. It has no effect on other
-                                socket types (e.g. datagram/UDP) and on processes
-                                unrelated to socket-based
+                                socket types (e.g. datagram/UDP) and
+                                on processes unrelated to socket-based
                                 activation. If the tcpwrap
                                 verification fails daemon start-up
                                 will fail and the connection is
                                 terminated. See
                                 <citerefentry><refentrytitle>tcpd</refentrytitle><manvolnum>8</manvolnum></citerefentry>
-                                for details.</para></listitem>
+                                for details. Note that this option may
+                                be used to do access control checks
+                                only. Shell commands and commands
+                                described in
+                                <citerefentry><refentrytitle>hosts_options</refentrytitle><manvolnum>5</manvolnum></citerefentry>
+                                are not supported.</para></listitem>
                         </varlistentry>
 
                         <varlistentry>
